js-runtime/index.js

```diff
@@ -924,20 +924,18 @@ module.exports = {
   seqrec: (seq, recurseFn) => [seq, recurseFn],
 
   // IO opcodes
-  httpget:  async url => {
+  httpreq:  async req => {
```

Comment:
much simpler impl haha

Reply:
Well, the opcode was based in part on this API, but this API is also very close to just generating the entire request string yourself and splatting onto a TCP socket.